app/controllers/api/v3/scenarios_controller.rb

```diff
@@ -88,7 +88,7 @@ def create
 
         @scenario = Scenario.new
 
-        if scaler_attributes && ! attrs[:descale]
+        if scaler_attributes && ! attrs[:descale] && ! Area.derived?(attrs.fetch(:area_code))
```

Comment:
> (scaler attributes being provided and/or the descale option).

Should it be possible to descale a scenario from a derived dataset? What would you even expect? To get a scenario with the `base_dataset` and the inputs scaled up to the size of Holland?